firefox-ios/Client/TabManagement/Legacy/LegacyTabManager.swift

```diff
@@ -905,24 +950,6 @@ class LegacyTabManager: NSObject, FeatureFlaggable, TabManager, TabEventHandler
         storeChanges()
     }
 
-    // Select the most recently visited tab, IFF it is also the parent tab of the closed tab.
-    private func selectParentTab(afterRemoving tab: Tab) -> Bool {
```

Comment:
This was only used in one place, so this logic is essentially checked in `updateSelectedTabAfterRemovalOf` (formerly called `updateIndexAfterRemovalOf`).